src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/TextEditingTargetNotebook.java

```diff
@@ -753,13 +878,50 @@ else if (data.getType() == RmdChunkOutputFinishedEvent.TYPE_INTERACTIVE &&
             // set dirty state if necessary
             setDirtyState();
          }
+
+         if (satelliteChunks_.contains(data.getChunkId()))
+         {
+            forwardEventToSatelliteChunk(
+               docUpdateSentinel_.getId(),
+               data.getChunkId(),
+               new ChunkSatelliteOutputFinishedEvent(
+                  ensureVisible,
+                  data.getScope()
+               )
+            );
+         }
       }
    }
 
+   @Override
+   public void onConsoleWriteOutput(ConsoleWriteOutputEvent event)
```

Comment:
@jmcphers found it! I believe we were also missing to do this: https://github.com/rstudio/rstudio/pull/765/commits/22856f7abd37304a98ed30e6e51ed9afe7a8a54c